ipatests/test_integration/test_replica_promotion.py

```diff
@@ -403,6 +403,7 @@ def test_replica_install_with_existing_entry(self):
             realm=replica.domain.name.upper())
         master.put_file_contents(ldif_file, entry_ldif)
         arg = ['ldapmodify',
+               '-ZZ',
```

Comment:
Perhaps out-of-scope but would it make sense to have ldapsearch and ldapmodify helpers to avoid all this duplication?

Reply:
+1